vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -17,6 +17,8 @@
        - ironic-endpoint-keepalived
        - ironic-log-watch
        - ironic-inspector-log-watch
+    become: yes
+    become_user: root
```

Comment:
Why do we need this ?

Reply:
When I did my local tests Ansible failed right here due to permissions, requiring me root permission. I know that it was working fine up until now without root, but I think it is not going to hurt. I can remove it and rerun the tests.

Reply:
ok, lets keep it then.